docs/src/pages/[platform]/components/card/examples/CardThemeExample.tsx

```diff
@@ -21,7 +21,7 @@ const cardTheme = {
 
 export const CardThemeExample = () => {
   return (
-    <AmplifyProvider theme={cardTheme}>
+    <ThemeProvider theme={cardTheme}>
```

Comment:
We should add a note in an Info alert above our Theme examples that the AmplifyProvider was renamed to ThemeProvider, and the ThemeProvider export is available since version 2.x.x (specific version added after we release?)

Reply:
Thanks that is a good call out, I've added an alert to all of the component examples and the theming getting started page.  For the version number I went with 2.18.3 because it looks like that will be our next release. But we can update that if we need to.